SPECS/kernel-uvm/config

```diff
@@ -920,26 +920,9 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_INET_ESP is not set
 # CONFIG_INET_IPCOMP is not set
 # CONFIG_INET_DIAG is not set
-CONFIG_TCP_CONG_ADVANCED=y
```

Comment:
potentially to follow up on the nature of this change

Reply:
This is to enhance network I/O performance